wasm/src/std.h

```diff
@@ -10,4 +10,18 @@
 void opa_abort(const char *msg);
 void opa_println(const char *msg);
 
+#ifdef DEBUG
+#include "printf.h"
+#define TRACE(...)                                   \
+    {                                                \
```

Comment:
Sorry, I wasn't clear enough. This looks OK but you should remove the outer curly brackets and then also remove the semi-colon after "while". Then you can use "TRACE();" in any context, like any statement.

Reply:
I figured there was some trick with the do-while inside the block, but using it as the wrapper makes way more sense 😆